src/libstd/collections/hash/map.rs

```diff
@@ -4076,11 +3262,10 @@ mod test_map {
     #[test]
     fn test_entry_take_doesnt_corrupt() {
         #![allow(deprecated)] //rand
-        // Test for #19292
+                              // Test for #19292
```

Comment:
uhh did you mean to do this

Reply:
I blame this entirely on rustfmt (I think it is trying to align the comment with the block above).

Reply:
cc @nrc @topecongiro